t/python/t0005-kvs.py

```diff
@@ -39,7 +40,9 @@ def set_and_check_context(self, key, value, type):
         kd2 = flux.kvs.KVSDir(self.f)
         nv = kd2[key]
         if isinstance(value, bytes) and type is str:
-            self.assertEqual(value.decode('utf-8'), nv)
+            self.assertEqual(value.decode("utf-8"), nv)
+        elif isinstance(value, bytes) and type is dict:
+            self.assertEqual(ast.literal_eval(value.decode("utf-8")), nv)
```

Comment:
Could use a comment -- at least, I don't understand exactly why `ast.literal_eval` is needed here.
Also instead of `self.assertEqual` should you use `self.assertDictEqual`?